nemo/collections/nlp/modules/common/megatron/transformer.py

```diff
@@ -240,7 +239,7 @@ def __init__(
                     ffn_hidden_size // get_tensor_model_parallel_world_size(), layernorm_epsilon
                 )
 
-    def forward(self, hidden_states):
+    def forward(self, hidden_states, infused_adapter=None):
```

Comment:
Best not to change forward. Why not simply add adapter capability to the attention module itself, and then let it handle all of it ?

Reply:
Discussed an approach with @titu1994 offline which I'll be implementing in a separate PR.